### ISSUE TITLE
The name prop is not applied to the CheckBox component.

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-react#526]
The name prop is not applied to the CheckBox component.

https://stackblitz.com/edit/react-wb7sc6?file=app/main.jsx

**Comment by simonssspirit** [comment]
As a workaround the ref can be used: 

````
<Checkbox label={'Unchecked state'} name='test'ref={ref => ref !== null && ref.element.current.setAttribute('name', 'test')}/>
````

**Comment by silviyaboteva** [comment]
Fixed in `v3.13.0-dev.202003310657`.
